bliss/encoder/image_normalizer.py

```diff
@@ -1,6 +1,10 @@
+# flake8: noqa: WPS348
 import warnings
+from pathlib import Path
 from typing import Dict
 
+import matplotlib.pyplot as plt  # noqa: WPS301
```

Comment:
```
from matplotlib import pyplot as plt
```

Reply:
plt is not used in this code if we delete the `plot_image` function which is designed for debug